src/FastTags/LexerTypes.hs

```diff
@@ -168,15 +175,11 @@ fixChar c
           OtherLetter           -> Just lower
           DecimalNumber         -> Just digit
           OtherNumber           -> Just digit
-          ConnectorPunctuation  -> Just symbol
-          DashPunctuation       -> Just symbol
-          OtherPunctuation      -> Just symbol
-          MathSymbol            -> Just symbol
-          CurrencySymbol        -> Just symbol
-          ModifierSymbol        -> Just symbol
-          OtherSymbol           -> Just symbol
           Space                 -> Just space
-          _other                -> Nothing
+          other                 ->
```

Comment:
Just FYI guard works in case branches too, e.g. other | Util.isSymbol ... -> ...